src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -1701,19 +1683,17 @@ class RideConstructionWindow final : public Window
             _currentTrackLiftHill |= CONSTRUCTION_LIFT_HILL_SELECTED;
         }
 
-        int32_t x;
         if ((IsTrackEnabled(TRACK_LIFT_HILL) && (_currentTrackCurve & RideConstructionSpecialPieceSelected) == 0)
             || (gCheatsEnableChainLiftOnAllTrack && currentRide->GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_HAS_TRACK)))
         {
             widgets[WIDX_CHAIN_LIFT].type = WindowWidgetType::FlatBtn;
-            x = 9;
         }
         else
         {
             widgets[WIDX_CHAIN_LIFT].type = WindowWidgetType::Empty;
-            x = 23;
         }
 
+        int32_t x = 45;
```

Comment:
Not sure I get this change either :'(

Reply:
it makes the ui wider and makes it easier to assign the very narrow curves to a symbol by inserting a new ui if the tracks have it. currently only my maurer has the 4 different curve radii active